README.md

```diff
@@ -4,11 +4,11 @@ Home of the Ntuplizer for the HGCAL reconstruction software studies
 
 Ntuple content definitions can be found at [Definitions.md](Definitions.md).
 
-This version is based on >= CMSSW_10_3_0_pre5, instructions below should provide you with the version for the TDR sample production. Please check if there are later `CMSSW_10_3_X` versions available to profit from bugfixes.
+This version is based on >= CMSSW_10_6_0_pre2, instructions below should provide you with the version for the TDR sample production. Please check if there are later `CMSSW_10_3_X` versions available to profit from bugfixes.
```

Comment:
The second sentence should have `CMSSW_10_6_X`instead of `CMSSW_10_3_X`

Reply:
Yes, it has escaped my `grep` :)
thanks for noticing